src/System.Private.CoreLib/src/System/Text/UTF8Encoding.cs

```diff
@@ -1315,7 +1315,7 @@ private static bool InRange(int ch, int start, int end)
             if (encoder != null)
             {
                 Contract.Assert(!encoder.MustFlush || ch == 0,
-                    "[UTF8Encoding.GetBytes] Expected no mustflush or 0 leftover ch " + ch.ToString("X2", FormatProvider.InvariantCulture));
```

Comment:
I have simplified this low-value message here because of it triggers a huge amounts of code to run during startup - that is pain for debugging.